contrib/netbox.env

```diff
@@ -0,0 +1,15 @@
+# Name is the Process Name
+#
+Name = 'Netbox'
+
+# ConfigPath is the path to the gunicorn config file.
+#
+ConfigPath=/usr/local/netbox/gunicorn.conf
+
+# WorkingDirectory is the Working Directory for Netbox.
+#
+WorkingDirectory=/usr/local/netbox/
+
+# PidPath is the path to the pid for the netbox WSGI
+#
+PidPath=/usr/local/netbox/netbox.pid
```

Comment:
I don't think PidPath is either necessary or desirable when using systemd

Reply:
Since gunicorn can create the pid, it is best to include it.

Reply:
Why? I mean:

1. Why is it a good idea to tell gunicorn to create a pidfile (just because it can?)
2. Why is it a good idea to tell systemd about the pidfile, when it won't read it?

The systemd `PIDFile=` setting is *only* supposed to be used with `Type=forking`, i.e. legacy software which daemonizes itself, and which cannot be told not to daemonize.

The preferred way of running processes under systemd is as directly spawned children which are supervised by it.

So if gunicorn doesn't fork (which is good), systemd will never read the pidfile.